README.md

```diff
@@ -38,17 +40,17 @@ Connect-Query is an expansion pack for [TanStack Query](https://tanstack.com/que
     - [SolidJS Example](#solidjs-example)
   - [What about Streaming?](#what-about-streaming)
 
-# Quickstart
+## Quickstart
```

Comment:
Most of this is duplicated in the docs. Are you sure it is a good idea to diverge?